pyazul/validate.py

```diff
@@ -15,32 +63,47 @@ def datavault_create(data):
 def datavault_delete(data):
     required = {
         'DataVaultToken': data.get('DataVaultToken', ''),
-        'TrxType': 'DELETE',
+        'TrxType': r'DELETE',
```

Comment:
@sjortiz could you share a reference for this one? This one is new for me :)

Reply:
in my editor DELETE and CREATE where being recognized as "keywords" maybe VSCode though it was an sql string. r in front of the prevents python from trying to parse it.

If this field isn't used or combined with nothing that will be parsed I think I can rollback that specific change. It doesn't do anything tho (I double checked and I think it's my editor going crazy)